pkg/controller/configmap/config.go

```diff
@@ -149,7 +134,11 @@ func getPrefixFromCIDR(cidr string) (uint32, error) {
 	return uint32(i), nil
 }
 
-func validateClusterNetwork(spec *configv1.NetworkSpec) []error {
+func (r *ReconcileConfigMap) validateClusterNetwork(spec *configv1.NetworkSpec) []error {
```

Comment:
I think it's better to split ReconcileConfigMap func and ReconcileConfigMapOc func into 2 config files. And maybe add interfaces for them?

Reply:
will try it.

Reply:
kept the original implementation of ReconcileConfigMap and additionally add interface Adaptor for it. But I didn't split ConfigMapK8s and ConfigMapOc which are implementing the Adaptor interface into 2 files. Do you have an example for the best practice?